api/src/main/java/org/openmrs/module/ugandaemrreports/data/converter/PersonAttributeDataConverter.java

```diff
@@ -24,8 +26,26 @@ public Object convert(Object obj) {
         if (obj == null) {
             return "";
         }
-
-        return ((PersonAttribute)obj).getValue();
+        String personAttribute = ((PersonAttribute)obj).getValue();
```

Comment:
@solemabrothers Rename this class to NationalityPersonalAttributeDataConverter to be very specific on its usage

Reply:
But it does not only look at Nationality alone, but it also looks at things like telephone numbers which are all person Attributes

Reply:
@solemabrothers you have other person attributes like marital status etc

Reply:
Renamed